### PR TITLE
Windows: Make Python binary work with runfiles symlink tree

### PR DESCRIPTION
**Pull request by meteorcloudy**

### CODE REVIEW

**Comment by laszlocsomor** [comment]
Could you take a look at the failing tests?

**Comment by meteorcloudy** [comment]
Failing tests have been fixed, PTAL~

File: src/test/shell/bazel/bazel_windows_example_test.sh [hunk 1]
```diff
@@ -196,6 +196,30 @@ function test_native_python() {
   assert_test_fails //examples/py_native:fail
 }
 
+function test_native_python_with_runfiles() {
+  BUILD_FLAGS="--experimental_enable_runfiles --build_python_zip=0"
+  bazel build -s --verbose_failures $BUILD_FLAGS //examples/py_native:bin \
+    || fail "Failed to build //examples/py_native:bin with runfiles support"
+  (
+  # Clear runfiles related envs
```
Comment (laszlocsomor) [inline src/test/shell/bazel/bazel_windows_example_test.sh]: To help readability, please indent the lines running in a subshell (line 204 .. 215).

File: src/test/shell/integration/runfiles_test.sh [hunk 2]
```diff
@@ -179,14 +179,15 @@ EOF
 
   # that accounts for everything
   cd ../..
-  # For shell binary, we build both `bin` and `bin.exe`, but on Linux we only build `bin`
-  # That's why we have one more symlink on Windows.
+  # For shell binary and python binary, we build both `bin` and `bin.exe`,
+  # but on Linux we only build `bin`.
+  # That's why we have two more symlinks on Windows.
```
Comment (laszlocsomor) [inline src/test/shell/integration/runfiles_test.sh]: Two more? But `bin.exe` is just one more.

Comment (meteorcloudy) [reply src/test/shell/integration/runfiles_test.sh]: We have one more for shell binary and one more for python binary.

File: src/test/shell/bazel/bazel_windows_example_test.sh (hunk 1 above)
Comment (meteorcloudy) [reply src/test/shell/bazel/bazel_windows_example_test.sh]: Done

**Pull request closed without merging**